Cloud tour story (#2987)
nasa-gibs/worldview: config/default/common/config/metadata/stories/clouds/step006.md

> This false color image (MODIS Bands 7-2-1) shows thick fog in the valleys northeast of Vancouver, Canada. The fog is a light white color in the center of the image, whilst the snow covered mountains are in blue/cyan and the vegetated areas are in green. Fog is essentially a cloud lying on the ground. It forms when the air reaches its dew point - where an air mass cools to a temperature that allows the water vapor to condense into water droplets. This image is from the Moderate Resolution Imaging Spectroradiometer (MODIS) instrument on board the Terra satellite on 21 February 2010.

Reviewer: Here is my attempt at rewriting this step to lead with a focus on fog.
```suggestion
Fog is essentially a cloud lying on the ground. Fog forms when the air reaches its dew point, that is, where an air mass cools to a temperature that allows the water vapor to condense into water droplets. This false color image (Bands 7-2-1) collected using the Moderate Resolution Imaging Spectroradiometer (MODIS) instrument on board the Terra satellite shows thick fog in the valleys northeast of Vancouver, Canada on 21 February 2010. The fog is a light white color in the center of the image, whilst snow covered mountains are in blue/cyan and vegetated areas are in green.
```
